yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -137,12 +142,12 @@ def decrypt(data):
         }, query=query, fatal=fatal)
         return api_json, decrypt
 
-    def _download_metadata(self, url, video_id, lang, props_key):
-        metadata = self._call_api(video_id, msg='metadata', query={'al': lang or 'ja'}, auth=False, fatal=False)
+    def _download_metadata(self, url, video_id, lang, props_keys):
```

Comment:
IMO this is the better way
```suggestion
    def _download_metadata(self, url, video_id, lang, *props_keys):
```

Reply:
Considering `prop_keys` aren't being spread when passed to travere_obj, I think current syntax is clearer.